pages/library/_viewId.vue

```diff
@@ -1,22 +1,22 @@
 <template>
   <div>
     <h1>{{ Name }}</h1>
-    <v-list>
-      <v-list-item
+    <div class="cardsContainer">
+      <v-card
         v-for="item in Items"
         :key="item.Id"
         :to="`../itemdetails/${item.Id}`"
+        class="card mt-5"
       >
-        <v-list-item-content>
-          <v-list-item-title>
-            {{ item.Name }}
-          </v-list-item-title>
-          <v-list-item-subtitle>
-            {{ item.ProductionYear }}
-          </v-list-item-subtitle>
-        </v-list-item-content>
-      </v-list-item>
-    </v-list>
+        <v-img class="cardImage" :src="imageLink(item.Id)" />
+        <v-card-title>
+          {{ item.Name }}
```

Comment:
```suggestion
          <span>{{ item.Name }}</span>
```

Reply:
The HTML ends up looking nicer if things like this are wrapped in span tags.

Reply:
Added